dds/idl/marshal_generator.cpp

```diff
@@ -635,6 +636,217 @@ namespace {
     }
   }
 
+  void gen_anonymous_sequence(const FieldInfo& sf)
+  {
+    if (!sf.as_act_->in_main_file()) {
+      if (sf.as_act_->node_type() == AST_Decl::NT_pre_defined) {
+        if (be_global->language_mapping() != BE_GlobalData::LANGMAP_FACE_CXX &&
+            be_global->language_mapping() != BE_GlobalData::LANGMAP_SP_CXX) {
+          be_global->add_include(("dds/CorbaSeq/" + nameOfSeqHeader(sf.as_act_)
+                                  + "SeqTypeSupportImpl.h").c_str(), BE_GlobalData::STREAM_CPP);
+        }
+      } else {
+        be_global->add_referenced(sf.as_act_->file_name().c_str());
+      }
+    }
+
+    const bool use_cxx11 = be_global->language_mapping() == BE_GlobalData::LANGMAP_CXX11;
+    const string check_empty = use_cxx11 ? "seq.empty()" : "seq.length() == 0";
+    const string get_length = use_cxx11 ? "static_cast<uint32_t>(seq.size())" : "seq.length()";
+    const string get_buffer = use_cxx11 ? "seq.data()" : "seq.get_buffer()";
+    if (use_cxx11) {
+      be_global->header_ << "struct " << sf.underscored_ << "_tag {};\n\n";
+    }
+
+    {
+      Function find_size("gen_find_size", "void");
```

Comment:
`find_size` and `max_marshaled_size` functions have been superseded by a new `serialized_size` function family that has different signatures. We probably need to discuss this more, but the primary functions in the new family are list listed at the top of `Serializer.h`.

Reply:
Should that kind of fix be part of this merge? Unless it was a bad merge that caused this issue, I think it might be best to do something like that post merge.

Reply:
If the merge result without that fix will not compile (or will fail tests), we should handle it here.

Reply:
I'm not sure about doing it in another PR because it's something that will keep the anonymous types test from compiling successfully.

Reply:
Did you mean `max_serialized_size`? Source generated from IDL seems to be wrong for calls to `max_serialized_size`.